src/pages/DetailsPage.tsx

```diff
@@ -116,43 +96,45 @@ function DetailsPage(props) {
 
     if (pronouns && pronouns.startsWith(CONST.PRONOUNS.PREFIX)) {
         const localeKey = pronouns.replace(CONST.PRONOUNS.PREFIX, '');
-        pronouns = props.translate(`pronouns.${localeKey}`);
+        pronouns = translate(`pronouns.${localeKey}` as TranslationPaths);
     }
 
     const phoneNumber = getPhoneNumber(details);
     const phoneOrEmail = isSMSLogin ? getPhoneNumber(details) : details.login;
     const displayName = PersonalDetailsUtils.getDisplayNameOrDefault(details, '', false);
 
-    const isCurrentUser = props.session.accountID === details.accountID;
+    const isCurrentUser = sessionAccountID === details.accountID;
 
     return (
         <ScreenWrapper testID={DetailsPage.displayName}>
-            <FullPageNotFoundView shouldShow={_.isEmpty(login)}>
-                <HeaderWithBackButton title={props.translate('common.details')} />
+            <FullPageNotFoundView shouldShow={!login}>
+                <HeaderWithBackButton title={translate('common.details')} />
                 <View style={[styles.containerWithSpaceBetween, styles.pointerEventsBoxNone]}>
                     {details ? (
                         <ScrollView>
                             <View style={styles.avatarSectionWrapper}>
+                                {/* @ts-expect-error TODO: Remove this once AttachmentModal (https://github.com/Expensify/App/issues/25130) is migrated to TypeScript. */}
                                 <AttachmentModal
                                     headerTitle={displayName}
                                     source={UserUtils.getFullSizeAvatar(details.avatar, details.accountID)}
                                     isAuthTokenRequired
                                     originalFileName={details.originalFileName}
                                 >
+                                    {/* @ts-expect-error TODO: Remove this once AttachmentModal (https://github.com/Expensify/App/issues/25130) is migrated to TypeScript. */}
                                     {({show}) => (
                                         <PressableWithoutFocus
-                                            style={[styles.noOutline]}
+                                            style={styles.noOutline as ViewStyle}
```

Comment:
Please [update](https://github.com/Expensify/App/pull/34368/files#diff-f9611c79d5f9fc6280f157d163451734dc39963b436ee1d521428216a591b3ff) `noOutline` typing instead
```suggestion
                                            style={styles.noOutline}
```

Reply:
updated